### PR TITLE
VR-3308 facilitate create config

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

**Comment by convoliution** [comment]
> 3. I have added the integration test test_config_file. It writes email and dev_key from env vars to a config file, deletes the env vars, creates a client object without arguments in constructor, creates a project, experiment, and dataset without arguments. Restores the env vars. Deletes the config.
>
> Shouldn't I delete this project and dataset? I think other tests don't delete them so I don't know how to do it.

That is a good idea. Most other tests delete projects and datasets automatically, but we will need to delete them manually here because we are manually creating a `Client` instead of [letting `pytest` automatically handling it](https://github.com/VertaAI/modeldb/blob/master/client/verta/tests/test_entities.py#L122).

This can be done with [`utils.delete_project(proj.id, conn)`](https://github.com/VertaAI/modeldb/blob/master/client/verta/tests/utils.py#L106) and [`utils.delete_datasets([dataset.id], conn)`](https://github.com/VertaAI/modeldb/blob/master/client/verta/tests/utils.py#L123).

> I don't know if something like this refactoring planned or not, but if not please consider it. It's not super easy, not super hard, but it's definitely a possible separate task.

You are right, the Client's test suite right now is more focused on integration than on unit tests, and many tests spend time communicating with ModelDB. There are many ideas for refactoring, including separating Client code into more focused pieces!

You may write unit tests here based on the functions you wrote (`_load_config()`, `_find_config()`). That would be appreciated!

File: client/verta/verta/client.py [hunk 1]
```diff
@@ -69,8 +70,10 @@ class Client(object):
 
     Parameters
     ----------
-    host : str
+    host : str, optional
         Hostname of the Verta Web App.
+    port: str, optional
+        Port of the Verta Web App.
```
[the remark this answers is not in the document]
Comment (evgeniilezhnin) [reply client/verta/verta/client.py]: fixed

**Comment by evgeniilezhnin** [comment]
> That is a good idea. Most other tests delete projects and datasets automatically, but we will need to delete them manually here because we are manually creating a `Client` instead of [letting `pytest` automatically handling it](https://github.com/VertaAI/modeldb/blob/master/client/verta/tests/test_entities.py#L122).
> 
> This can be done with [`utils.delete_project(proj.id, conn)`](https://github.com/VertaAI/modeldb/blob/master/client/verta/tests/utils.py#L106) and [`utils.delete_datasets([dataset.id], conn)`](https://github.com/VertaAI/modeldb/blob/master/client/verta/tests/utils.py#L123).

fixed

**Comment by evgeniilezhnin** [comment]
> You are right, the Client's test suite right now is more focused on integration than on unit tests, and many tests spend time communicating with ModelDB. There are many ideas for refactoring, including separating Client code into more focused pieces!
> 
> You may write unit tests here based on the functions you wrote (`_load_config()`, `_find_config()`). That would be appreciated!

This will need some refactoring. I think it will need some time spent, maybe 4 hours. Could you make it a separate task in Jira please assign to me and set priority, please? Because I have a high priority task at the moment.

**Pull request merged**